docs/index.md

```diff
@@ -82,19 +82,17 @@ This allows you to rely on UI5 Tooling for UI5-specific build functionality and
 All available APIs are documented in the [UI5 Tooling API Reference](https://sap.github.io/ui5-tooling/v3/api/index.html).
 
 ```js linenums="1"
-const {normalizer} = require("@ui5/project");
-const {builder} = require("@ui5/builder");
+import {graphFromPackageDependencies} from "@ui5/project/graph";
```

Comment:
I think it's fair to assume that many developers still work with CommonJS (just like we do with many of our internal scripts). Maybe we can put both flavors into respective [content tabs](https://squidfunk.github.io/mkdocs-material/reference/annotations/#in-content-tabs) (e.g. `ESM` and `CommonJS`)?

Same for the custom task/middleware examples.

Code looks fine 👍

Reply:
Good idea! Will do that! And also provide a sample code in https://github.com/SAP/openui5-sample-app/tree/demo-server-middleware-extensibility-v3

Reply:
Done

Reply:
But then we should still put **ESM** as the first tab, right?

Reply:
Good point. I'll rearrange the samples

Reply:
Done